### ISSUE TITLE
imageUrlBuilder baseUrl does not exist in type

### ISSUE THREAD

**Issue by PaulieScanlon** [sanity-io/image-url#66]
**Describe the bug**
When using `baseUrl` with imageUrlBuilder i see a type definition warning telling me the `baseUrl` type does not exists. 
When using `baseUrl` my image URL isn't constructed correctly and I see the broken image link in the browser.

Reproduction repo here: [netlify-astro-sanity-portable-text](https://github.com/PaulieScanlon/netlify-astro-sanity-portable-text/blob/main/src/components/image.astro)

**To Reproduce**
Here's my config:

```javascript
import imageUrlBuilder from '@sanity/image-url';

const builder = imageUrlBuilder({
  baseUrl: import.meta.env.PUBLIC_SITE_URL,
  projectId: import.meta.env.PUBLIC_SANITY_STUDIO_PROJECT_ID,
  dataset: import.meta.env.PUBLIC_SANITY_STUDIO_DATASET,
});

And here's the typescript warning/erorr

> Object literal may only specify known properties, and 'baseUrl' does not exist in type 'SanityClientLike | SanityProjectDetails | SanityModernClientLike'.ts(2353)
(property) baseUrl: any

```

**Expected behavior**
This should render an image.

**Additional context**
If i replace `baseUrl` with `url` i still get TypeScript errors, but the image URL is constructed correctly and the image renders in the browser. 

E.g

```diff
import imageUrlBuilder from '@sanity/image-url';

const builder = imageUrlBuilder({
-  baseUrl: import.meta.env.PUBLIC_SITE_URL,
+  url: import.meta.env.PUBLIC_SITE_URL,
  projectId: import.meta.env.PUBLIC_SANITY_STUDIO_PROJECT_ID,
  dataset: import.meta.env.PUBLIC_SANITY_STUDIO_DATASET,
});


```

**Comment by PaulieScanlon** [comment]
Hey, false alarm i see where i messed up, thanks @ChrisLaRocque!

Ok. so in the docs under [Usage](https://www.sanity.io/docs/image-url#usage) it shows a snippet like this.

```javascript
import myConfiguredSanityClient from './sanityClient'
import imageUrlBuilder from '@sanity/image-url'

const builder = imageUrlBuilder(myConfiguredSanityClient)
```

There's no mention of what `myConfiguredSanityClient` is, and if you click the link around [your configured sanityClient](https://www.npmjs.com/package/@sanity/client), you're taken to the npm package [README](https://www.npmjs.com/package/@sanity/client) which explains how to configure the sanityClient.

But that's not how you configure the sanityClient using the Astro integration, that's all handled in `astro.config.mjs`

E.g 

```javascript
export default defineConfig({
  output: 'hybrid',
  integrations: [
    sanityIntegration({
      projectId: 'your-project-id',
      dataset: 'your-dataset-name'
      useCdn: false,
      studioBasePath: '/admin',
    }),
    react(),
  ],
});
```

Could i propose a change to the docs, maybe add code snippet for configuring the `imageUrlBuilder`

E.g 

```javascript
import imageUrlBuilder from '@sanity/image-url';

const builder = imageUrlBuilder({
  projectId: 'your-project-id',
  dataset: 'your-dataset-name'
});
```

Might be worth combing every page in your docs because a lot of what you're talking about only really applies to Next.js. Now that Astro is around there might be a need to add some caveats explaining the differences when using the Astro integration?

**Comment by rexxars** [comment]
Fixed in v1.1.0
